File: c/006/006.c
#include <stdio.h>

int main(int argc, char *argv[]) {
  unsigned long long sumOfSquares = 0;

  unsigned long long sum = 0;

  unsigned long long squareOfSum = 0;

  for (unsigned long long i = 1; i <= 100; i++) {
    sumOfSquares += (i * i);

    sum += i;
  }

  squareOfSum = sum * sum;

  printf("%llu\n", squareOfSum - sumOfSquares);

  return 0;
}

File: lua/006/006.lua
sum_of_squares = 0

sum = 0

for i = 1, 100, 1 do
   sum_of_squares = sum_of_squares + (i * i)

  sum = sum + i
end

square_of_sum = sum * sum

print(square_of_sum -sum_of_squares)
